src/components/MediaPlayer/VideoJS/components/js/VideoJSProgress.js

```diff
@@ -440,6 +436,27 @@ function ProgressBar({
     }
   };
 
+  /**
+   * Set start values for progress bar
+   * @param {Object} cTimes canvas start and end times
+   */
+  const initializeProgress = (cTimes) => {
+    let {start, customStart} = cTimes
+    if (customStart > start) {
+      setProgress(customStart);
+      setInitTime(customStart);
+
+      setCurrentTime(customStart);
+      player.currentTime(customStart);
+    } else {
+      setProgress(start);
+      setInitTime(start);
+
+      setCurrentTime(start);
+      player.currentTime(start);
+    }
+  }
+
```

Comment:
Maybe we could evaluate the condition in the parent function and pass down either `customStart` or `start`to `initializeProgress()` function to setup the initial state without duplicating the setter methods in `initializeProgress` method?